styles/src/navigation/navigation.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { makeStyles, shorthands, tokens } from "@fluentui/react-components";
+
+/**
+ * How to use:
+ * ```
+ * <div className={navStyles.container}>
+ *   <TabList
+ *     className={navStyles.tabList}
+ *     size="large"
+ *     vertical
+ *   >
+ *     <Tab />
+ *     <Tab />
+ *     <div className={navStyles.spacer}/>
+ *     <Tab />
+ *   </TabList>
+ * </div>
+ * ```
+ */
+export const useNavigationStyles = makeStyles({
+  container: {
+    ...shorthands.padding(
+      tokens.spacingVerticalXXL,
+      tokens.spacingHorizontalS,
+      tokens.spacingVerticalL
+    ),
+    backgroundColor: tokens.colorNeutralBackground4,
+  },
+  spacer: {
+    flexGrow: 1,
+  },
+  tabList: {
```

Comment:
On a second look, this part (rowGap) is not needed.
It is included in the tablist styles (recently pushed by Jimmy)

Reply:
I didn't want to import three different styles to create a navigation component. 
Actually I don't think we need the `useTabListStyles` to create a navigation component. Because the TabList component has already `flex` and we can just use `vertical` prop intead of using `useTabListStyles`. And for other styles, we can import from this styles.

So I used just `useTabStyles` and this navigation style in ADMX and IAM.